### DISCUSSION TITLE
`cargo run` or `script/bundle` failed: 'invalid version number'

### DISCUSSION THREAD

**Discussion by laojianzi** [zed-industries/zed#6975] [Feature Requests]
Macos 11.6.2 (20G314)
Xcode 13.2.1 (13C100)
Apple clang version 13.0.0 (clang-1300.0.29.30)

```shell
warning: `live_kit_client` (build script) generated 1 warning
   Compiling live_kit_client v0.1.0 (*/zed/crates/live_kit_client)
error: failed to run custom build command for `live_kit_client v0.1.0 (*/zed/crates/live_kit_client)`

Caused by:
  process didn't exit successfully: `*/zed/target/debug/build/live_kit_client-c10cade18b340a4d/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=MACOSX_DEPLOYMENT_TARGET
  cargo:rerun-if-changed=LiveKitBridge/Sources
  cargo:rerun-if-changed=LiveKitBridge/Package.swift
  cargo:rerun-if-changed=LiveKitBridge/Package.resolved
clang: error: invalid version number in '-target x86_64-apple-macosx10.15.710.10'

...

  --- stderr
  thread 'main' panicked at crates/live_kit_client/build.rs:77:9:
  Failed to compile swift package in */zed/crates/live_kit_client/LiveKitBridge
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by tml** [comment]
What does `clang --version` report?

**Reply by laojianzi** [reply]
Apple clang version 13.0.0 (clang-1300.0.29.30)
Target: x86_64-apple-darwin20.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
